src/lib/installer.ts

```diff
@@ -0,0 +1,225 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import { Octokit } from '@octokit/core'
+import { resolve } from 'path'
+
+const octokit = new Octokit()
+
+/**
+ * Gets path to binary.
+ * 
+ * @param tag Optional release tag. E.g. `v1.15.0`
+ */
+export function getBinPath(tag?: string): string | undefined {
+	let binpath = tag ? path.join(__dirname, '../..', `bin/bin_${tag}`) : path.join(__dirname, '../..', `bin/bin_current`)
+	let exists = fs.existsSync(binpath)
+
+	if (exists)
+		return binpath
+	else {
+		binpath = tag ? path.join(__dirname, '../..', `bin/bin_${tag}.exe`) : path.join(__dirname, '../..', `bin/bin_current.exe`)
+		exists = fs.existsSync(binpath)
+
+		if (exists)
+			return binpath
+	}
+
+	return undefined
+}
+
+/**
+ * Gets path to binary asynchronously.
+ * 
+ * @param tag Optional release tag. E.g. `v1.15.0`
+ */
+export async function getBinPathAsync(tag?: string): Promise<string | undefined>
+export async function getBinPathAsync(_tag?: string) {
```

Comment:
I cannot see any references to this function.

Reply:
It's exported.